supervisor/api/supervisorhttp/parser_test.go

```diff
@@ -0,0 +1,128 @@
+package supervisorhttp
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// parse parses the command from JSON into a "command"
+// struct with a check for success.
+func parse(t *testing.T, jsonByte []byte, cmd *command) {
+	err := parseCommand(bytes.NewReader(jsonByte), cmd)
+	assert.Nilf(t, err, `Can't parse a command: "%v".`, err)
+}
+
+// parse parses the command from JSON into a "command"
+// struct with a check for failure.
+func parseFail(t *testing.T, jsonByte []byte) {
```

Comment:
I like such function to be named `checkParseFail` or `assertParseFails`.

Reply:
If you like the name `assertParseFails`, I don't mind calling it that.